index.html

```diff
@@ -0,0 +1,1284 @@
+    <link rel="stylesheet" href="assets/reset.css" />
+    <link rel="stylesheet" href="assets/style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:wght@400&display=swap"
+      rel="stylesheet"
+    />
+    <title>Newsweek</title>
+  </head>
+
+  <body>
+    <header>
+      <div class="top-title">
+        <div class="weather">
+          <h1>
+            <span
+              class="iconify"
+              data-icon="mdi:weather-partly-cloudy"
+              data-inline="false"
+            ></span>
+            20° Terrassa, ES >
+          </h1>
+          <p>Fri, Oct 23, 2020</p>
+        </div>
+
+        <div class="logo">
+          <img src="assets/img/logo.jpg" alt="logo" />
+        </div>
+
+        <div class="login">
+          <img src="assets/img/login.jpg" alt="login" />
+        </div>
+      </div>
+
+      <nav>
+        <div class="nav-logo">
+          <img src="assets/img/smallogo.jpg" alt="logo" />
+        </div>
+        <div class="nav-login1">
+          <img src="assets/img/login2.jpg" alt="login" />
+        </div>
+      </nav>
+
+      <div class="first-nav">
+        <div class="nav-logo">
+          <img src="assets/img/smallogo.jpg" alt="logo" />
+        </div>
+        <div class="nav-login1">
+          <img src="assets/img/login3.jpg" alt="login" />
+        </div>
+      </div>
+
+      <div class="nav2">
+        <ul>
+          <li>U.S.</li>
+          <li>World</li>
+          <li>Business</li>
+          <li>Tech & Science</li>
+          <li>Culture</li>
+          <li>Newsgeek</li>
+          <li>Sports</li>
+          <li>Health</li>
+          <li>The Debate</li>
+          <li>Vantage</li>
+          <li>Weather</li>
+        </ul>
+        <input type="search" placeholder="Search" />
+        <span class="iconify" data-icon="ei:search" data-inline="false"></span>
+        <img src="assets/img/pesta.jpg" alt="s" />
+      </div>
+
+      <div class="nav3">
+        <ul>
+          <li>TRENDING</li>
+          <li>DONALD TRUMP</li>
+          <li>JOE BIDEN</li>
+          <li>STIMULUS PACKAGE</li>
+          <li>CORONAVIRUS</li>
+          <li>AMY CONEY BARRETT</li>
+          <li>2020 ELECTION</li>
+          <li>COUNTRIES THAT COUNT</li>
+        </ul>
+      </div>
+    </header>
+
+    <main>
+      <section>
+        <div class="container-fluid my-container">
+          <div class="row my-row1">
+            <div class="col-12 col-md-12 col-lg-8">
+              <div class="row">
+                <div
+                  class="col-12 col-md-6 order-2 order-md-1 col-lg-5 my-col1"
+                >
+                  <h1>FEATURED STORIES</h1>
+
+                  <div class="featured-stories">
+                    <img
+                      src="assets/img/hunter-biden-tony-bobulinski-wall-street-journal.webp"
+                      alt="biden"
+                    />
+                    <h2>U.S.</h2>
+                    <h3>
```

Comment:
Please add the Navigation Link. (Please add the Navigation Link where we are using the pointer. Update the navigation link on the complete page)